### PR TITLE
✨ indicator tags (lucas)

### PR DESCRIPTION
**Pull request by lucasrodes**
Part of https://github.com/owid/etl/issues/1868

### CODE REVIEW

**Review by pabloarosado — APPROVED** [review]
I've left a few comments and suggestions. More importantly, there are two things I'm not sure about:
* There are multiple cases where you have simply replaced `description` -> `description_short`, even when the description is too long. If anything, use the first point of `description_key` (or `description_from_producer` or `description_processing`, when relevant).
* You have added topic tags even to steps that use the old metadata. I don't know if this can cause some issues, but, more importantly, I think this signals that those steps "have been adapted to the new metadata", while they haven't properly been adapted. So I'd prefer to skip those files for now.

File: etl/steps/data/garden/demography/2023-06-12/population_density.meta.yml [hunk 1]
```diff
@@ -6,13 +12,13 @@ tables:
     variables:
       population_density:
         title: Population density
-        description: |
+        description_short: |
           Population density estimated by Our World in Data using population estimates from multiple sources
           and land area estimates by the Food and Agriculture Organization of the United Nations. We obtain it
           by dividing the population estimates by the land area estimates.
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply etl/steps/data/garden/demography/2023-06-12/population_density.meta.yml:18]: Good point. I've actually decided to move all this text to `description_processing`. And use an actual definition of the indicator in `description_short`.

File: etl/steps/data/garden/war/2023-09-21/cow.meta.yml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 definitions:
+  common:
+    presentation:
+      topic_tags:
+        - War & Peace
   all:
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply etl/steps/data/garden/war/2023-09-21/cow.meta.yml:6]: I have some definitions in the YAML file, that I don't want to propagate to indicators' metadata, but only use with explicit references (`*`).

File: etl/steps/data/garden/war/2023-01-18/bouthoul_carrere_1978.meta.yml [hunk 3]
```diff
@@ -1,3 +1,10 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - War & Peace
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply etl/steps/data/garden/war/2023-01-18/bouthoul_carrere_1978.meta.yml]: ive just removed this for old metadata files

File: etl/steps/data/garden/un/2023-08-02/comtrade_pandemics.meta.yml [hunk 4]
```diff
@@ -17,130 +24,145 @@ tables:
...
+        description_key:
+          - |-
+            Products included are: "Breathing appliances and gas masks", "Gloves, mittens and mitts (knitted or crocheted)", "Gloves, mittens and mitts (not knitted or crocheted)",
+            "Vulcanized (other than hard rubber), gloves other than surgical gloves.",  "Plastics in clothing",  "Spectacles, goggles and the like",  "Surgical gloves", and
+            "Textiles and made up articles".
+      ort_cif_breathing_appliances_per_capita:
         title: Imports of breathing appliances and gas masks (per capita)
         unit: "current US$ per capita"
-        description: >-
+        description_short: |-
```
Comment (pabloarosado) [inline etl/steps/data/garden/un/2023-08-02/comtrade_pandemics.meta.yml:107]: I just noticed you use "|-" here but then have a line break in the middle of a sentence. I suppose you should use ">-" instead.

**Review by pabloarosado — APPROVED** [review]
Looks good! I left a minor suggestion and fixed a couple of issues. Apart from that, once CI works we can merge it.

**Pull request merged**